Automattic/wp-calypso
# Happiness: add Upwork fr support

#### Changes proposed in this Pull Request

* Add `fr` Upwork support

#### Testing instructions

- Ensure that Happychat has availability (by default development Calypso connects to staging Happychat)
- Sign into Calypso as a user that has at least one paid upgrade (like a Theme, or a paid plan) but does not have any site with a Business or eCommerce plan.
- Make sure your language setting is not a Spanish locale or French locale.
- Check the inline "Contact Us" form — it should be directing you to chat support ("Chat with us" on the button)
- Now change your language setting to `fr-ca` (French Canadian).
- Re-open the inline "Contact Us" form — it should now direct you to email support even though Happychat is still available.
- Try the same thing with a user that has a Business or eCommerce plan — they should receive Chat support regardless of language setting.
